src/test/java/io/strimzi/kafka/bridge/http/service/BaseService.java

```diff
@@ -0,0 +1,19 @@
+package io.strimzi.kafka.bridge.http.service;
+
+import io.vertx.ext.web.client.WebClient;
+
+
+public class BaseService <B extends BaseService> {
```

Comment:
what's the purpose of this generic type B here? The same question for `ConsumerService` and `TopicService`, it seems you don't use these generic types (B, C, and T) in the tests.

Reply:
oh, I forget to remove it from `BaseService`. We no need to have it. But as for `ConsumerService` and `TopicService` I use generics to have DSL for configuring and sending the requests